src/util/findIssueKeys.ts

```diff
@@ -16,22 +16,22 @@
  *      - JRA 123 (missing hyphen from key)
  */
 
-export const issueKeyRegex = (str: string) => {
-    if (typeof str !== 'string') return null;
+export const issueKeyRegex = (str: string): string[] => {
+    if (typeof str !== 'string') return [];
```

Comment:
replaced `null` as a return value with `[]` in all the places since that seems to be the more useful return value